asyncapi.http.kafka.proxy/docker/compose/docker-compose.yaml

```diff
@@ -1,16 +1,12 @@
 version: '3'
 services:
   zilla:
-    image: ghcr.io/aklivity/zilla:${ZILLA_VERSION}
+    image: ghcr.io/aklivity/zilla:develop-SNAPSHOT
     container_name: zilla
-    pull_policy: always
```

Comment:
pull_policy is needed when the build isn't a local build. probably best to just comment it out in the future.